### PR TITLE
refactor: main alerts and modals

### PR DESCRIPTION
**Pull request by Tanya-atatakai**
## Proposed changes

update alerts and modals on main

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: frontend/components/MainPage/sections/GasBalanceSection.tsx [hunk 1]
```diff
@@ -100,12 +96,21 @@ const TooltipContent = styled.div`
   }
 `;
 
-type GasBalanceSectionProps = { serviceConfigId: string };
```
Comment (Tanya-atatakai) [inline frontend/components/MainPage/sections/GasBalanceSection.tsx:103]: remove this from here as its only used deeper in `BalanceStatus` but lmk if there was another plan for it

File: frontend/components/MainPage/sections/GasBalanceSection.tsx [hunk 2]
```diff
@@ -116,23 +121,24 @@ export const GasBalanceSection = ({
     >
       <Text type="secondary">
         Trading balance&nbsp;
-        {serviceSafes.length > 0 && (
+        {serviceSafe && (
```
[the remark this answers is not in the document]
Comment (truemiller) [reply frontend/components/MainPage/sections/GasBalanceSection.tsx:124]: its a product abstraction, services are auto funded from master safe
staging is correct

**Review by truemiller — APPROVED** [review]
looks good, just slight comment on your comment regarding staging x master safe

File: frontend/components/MainPage/sections/GasBalanceSection.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply frontend/components/MainPage/sections/GasBalanceSection.tsx]: updated to master safe

**Pull request merged**